### ISSUE TITLE
Update build procedures in README

### ISSUE THREAD

**Issue by joewiz** [eXist-db/website#1]
The build procedures should be brought up to date. Some things that look out of date:

- prerequisites: eXist version
- building: ant dependency appears to not be required at all
- building: who is "I"?

**Comment by adamretter** [comment]
Could switch to the Maven and Claudius's Kuberam EXPath plugin to make building and releasing simpler. I already use that for the Versioning Module, XProc Module, and CQL Module.

**Comment by ljo** [comment]
Well for prereqs it is probably still true even if it looks dated. Otherwise I just removed the most blatantly silly stuff.

**Comment by joewiz** [comment]
@ljo Ah, okay, I retract the comment about the prereqs then. As to building, though, I didn't see a `build.xml` file in source, so are the ant-related instructions still relevant?

**Comment by joewiz** [comment]
I just sat down to build this app to review @adamretter's PR #14, but while I have successfully run `npm install` and `bower install`, there is no `build.xml` file, so the instructions in the readme to run `ant local-xar` do not work. After nearly giving up, I thought I'd try the following the "building with grunt" directions to see if they worked, so I [installed gruntjs](https://gruntjs.com/getting-started) and ran `grunt`. This successfully deposited `homepage-3.0.3.zip` in my the `build` subdirectory. I renamed this file as `homepage-3.0.3.xar` and installed the package via the Dashboard Package Manager. 

I would welcome any work either updating the documentation with the official build procedures, or updating the build procedures themselves.

**Comment by adamretter** [comment]
@joewiz I had exactly this problem earlier in the week. I also noted that the content in the repo doesn't necessarily match what is live on exist-db.org which is a sad thing. I asked @JoernT at eXist Solutions if they could find some time to fix this, as I took a look but I have no idea how this works.

**Comment by joewiz** [comment]
@JoernT Are there any tasks remaining on this front? I'm happy to help but am not sure what tasks remain. For example, I see the new directions say, simply, run `ant`, but in fact, version numbers have to be updated in various places (the .tmpl files aren't being used by the build file AFAICS). So presumably one task could be to update build.xml to use centrally-defined build properties. Similarly, are there outstanding code or documentation tasks with regard to the grunt and bower files stored here in the repo?

**Comment by adamretter** [comment]
We could move this over to the Maven build instead of Ant, so that version numbers etc are all handled auto-magically.

**Comment by line-o** [comment]
Is this still an issue? @joewiz

**Comment by joewiz** [comment]
@line-o Indeed, reviewing the repository as it is today, it remains the same as I described noted in my Jan 2018 post above. The Ant build.xml file does compress the repository into a .xar file, but the archive contains artifacts which are typically touched during the build process: 

1. The .tmpl files suggest that there is some process for applying package metadata to the package descriptor files (expath-pkg.xml and repo.xml), but the Ant build process doesn't include any facility for doing so. There is no build.properties (or variant thereof). 
2. The presence of the bower.json and gruntfile.js files indicates that Bower and Grunt were used to perform build tasks, like fetching dependencies, but there is no mention of these procedures in the README file or description of how they relates to the existing advice to run `ant`. 
3. The Travis facility is inactive, so the "passing" badge in the README is out of date. 

The app "works fine," and we have been using it without major issue to update the exist-db.org website, but the procedure in the README doesn't address how to build the package in a manner that updates package descriptors or fetches/updates dependencies.

If we have any resources to dedicate to addressing these issues, one route would be to (a) update the Ant file to update the package descriptors and (b) document how the bower and grunt steps for fetching/updating dependencies are performed. Another route would be to modernize the package's dependency management and build procedures. This would probably take the form of steps like:

- deleting the ant, bower, grunt, and travis artifacts
- declaring dependencies in an npm package.json file
- fetching dependencies via npm
- switching the build from using ant to using exist-generator (which gives us a nice README and GitHub Actions integrations out of the box) or the maven archetype
- testing CI and updating the README with the new build procedures
